app/assets/stylesheets/custom.css.scss

```diff
@@ -85,3 +87,25 @@ a:hover, a:active {
     background-color: $shrimp-pink;
     border-color: $shrimp-pink;
 }
+
+.polaroid {
+    position: relative;
+}
+
+.polaroid img {
+    border: 10px solid #fff;
+    border-bottom: 45px solid #fff;
+    -webkit-box-shadow: 3px 3px 3px #777;
+    -moz-box-shadow: 3px 3px 3px #777;
+    box-shadow: 3px 3px 3px #777;
```

Comment:
Color literals like `#777` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces